server/schemas/project.graphql

```diff
@@ -12,6 +12,7 @@ enum ProjectPublicationScope {
 type ProjectPublication {
   scope: ProjectPublicationScope!
   assetPublic: Boolean!
+  token: String
```

Comment:
```suggestion
```
For security reasons, it's better to reveal the token **only** once it is generated. However, we need to confirm this with Nova first.